meshapp/src/Routing/FooterRoutes.tsx

```diff
@@ -0,0 +1,31 @@
+import { Routes, Route } from "react-router-dom";
+import { paths } from "./RoutePaths";
+import LoggedInFooter from "../components/Footer/LoggedInFooter";
+import LoggedOutFooter from "../components/Footer/LoggedOutFooter";
+
+//Paths to render in either the logged in header or logged out footer
+const LOGGED_IN_PATHS = [
+  paths.logged_in_home,
+  paths.profile_page,
+  paths.profile_swipe,
+];
+const LOGGED_OUT_PATHS = [
+  paths.logged_out_home,
+  paths.login_page,
+  paths.forgot_password,
+  paths.sign_up,
+];
+
+//contains routes for the footer
+export default function FooterRoutes() {
+  return (
+    <Routes>
+      {LOGGED_IN_PATHS.map((path) => (
+        <Route path={path} element={<LoggedInFooter />} />
+      ))}
```

Comment:
We have different footers based on login?

Reply:
The footer with the message icon is different from the one with all the links right? That's what I'm seeing on the figma

Reply:
The message icon shouldn't be a part of the footer it should be by itself

Reply:
It's shown consistently along all the logged in pages at the bottom though so shouldnt it be a footer? should the footer with the links be included in the logged in pages as well?

Reply:
If only one thing is different, the whole footer shouldn't be rendered as a different footer

Reply:
Its showed down there. It should be like the facebook message bar

Reply:
So is it meant to be something that is fixed at the bottom and appears over other items similarly to the 2FA banner? In what other ways should I be displaying this for the logged in pages?